NebulaHost/PacketProcessors/GameHistory/GameHistoryDataRequestProcessor.cs

```diff
@@ -0,0 +1,29 @@
+﻿using LZ4;
+using NebulaModel.Attributes;
+using NebulaModel.Networking;
+using NebulaModel.Packets.GameHistory;
+using NebulaModel.Packets.Processors;
+using System.IO;
+using System.IO.Compression;
+
+namespace NebulaHost.PacketProcessors.GameHistory
+{
+    [RegisterPacketProcessor]
+    public class GameHistoryDataRequestProcessor : IPacketProcessor<GameHistoryDataRequest>
+    {
+        public void ProcessPacket(GameHistoryDataRequest packet, NebulaConnection conn)
+        {
+            using (MemoryStream ms = new MemoryStream())
+            using (MemoryStream ms2 = new MemoryStream())
```

Comment:
why 2 memory stream?

Reply:
right, I was initially exporting two separate objects and sending them in the packet. And during the cleanup, I forgot to remove it.